packages/kit/package.json

```diff
@@ -8,7 +8,8 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
-      "import": "./dist/index.mjs"
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.mjs"
```

Comment:
This is unsafe especially that i constantly see modules wrongly build with `.cjs` in dist and actually do `require('@nuxt/kit'`) but seems we don't have any choice at least until patched versions of nuxt 3 are widely adapted...

Reply:
oh, this is terrible, I agree. But I think `require` shouldn't work in any case, as we have `type: "module"`